examples/hpc/dla-driver/examples/depthwise.rs

```diff
@@ -0,0 +1,99 @@
+#![no_std]
+#![no_main]
+
+#[macro_use]
+extern crate alloc;
+
+use headsail_bsp::{init_alloc, rt::entry, sprint, sprintln};
+use panic_halt as _;
+
+use dla_driver::tensor3::{Order3, Tensor3};
+use dla_driver::tensor4::{Order4, Tensor4};
+
+use alloc::vec::Vec;
+
+fn conv_test() {
+    sprintln!("conv_test: enter");
+    let din: Vec<i8> = vec![
```

Comment:
Tag this entry with `#[rustfmt::skip]` to make sure it doesn't get autoformatted.

Reply:
Tagged